### DISCUSSION TITLE
my application keeps getting rejected

### DISCUSSION THREAD

**Discussion by dephacto** [community/community#135243] [GitHub Education]
### Hi everyone,

My application keeps getting rejected and I don't know why
Here is my organization verification document
![schoolihab](https://github.com/user-attachments/assets/ea93890b-bdcf-4773-bf9a-0b4cc0cae5bd)

**Comment by TheGiraffe3** [comment]
Maybe because it says 2021/2022?

**Comment by dephacto** [comment]
I submitted another one with 2023/2024. Also rejected it

**Comment by PakmanGames** [comment]
Here's a few things you can try:

- Make sure the name on your enrollment letter is the same as your name in your profiles billing info settings
- Take a better photo (the rejection could have been automatic by a robot)
